cdk/visit/lambda_code/test_api/test_api.py

```diff
@@ -84,14 +84,17 @@ def handle_test_api(self):
             "last_updated":(unix_timestamp_for_ttl)
         }
 
-        register_data = json.dumps(register_data)
+        register_data = json.dumps(register_data_dict)
 
         reg_response = http.request('POST', str(api_url)+"register",body=register_data)
 
         if reg_response.status != 200: 
             raise Exception("Register API Call Failed")
 
 
+        print("Canary Successful at " + str(dt_string) + " for username: " + str(register_data_dict["username"]))
```

Comment:
I think the cloud watch logs will include the time stamp so there’s no need to bake it in here. Can you check and amend this PR if the time stamps are already available?

I also want to call out that it is very important to be careful when considering logging customer data. Logging a request id or a hash might be good ways to accomplish the same thing without making your logs as sensitive as your database. Access to the logs is already restricted, but it’s good to be conscientious about what kind of data you put where. 

Can you talk with Prof. Russell about your plan for user checkin system’s data classifications and privacy policy? I’d be happy to hop on a call and provide some general advice but this is ultimately up to y’all to lead.

Reply:
I have adjusted the commit message. Also, Cloudwatch does have the timestamp, so I added a new commit to remove that from the print statement. 

With regards to the username, this log will only take place from the TestAPI lambda which generates its own dummy requests and usernames. The username referenced in that print statement will only be the username of a Canary test item. This should never touch the actual customer data and the only thing in the logs would be the Canary's username (in the form CANARY_TEST_<SOME_DATETIME>) which will be deleted after a few minutes from Dynamo ttl. 

I didn't believe that this information would be sensitive or need to be restricted/hashed, but let me know if you think we still might want to restrict that data.

Reply:
Logging the canary test username will certainly be fine - I did not understand that context.